Build fails for Swift 5.7.0

Based on `swift-http-types`'s `Package.swift` this package support Swift `5.7.0+`. However if I try to build it using Swift `5.7.0` it fails. Swift `5.7.1` works however.

See Swift `5.7.0` logs:

```bash
➜  swift-web-push git:(ff-fix-concurrency-problem) ✗ docker run -v $(pwd):/src -w /src -it swift:5.7.0-jammy /bin/bash
Unable to find image 'swift:5.7.0-jammy' locally
5.7.0-jammy: Pulling from library/swift
56a2caa6b2c6: Pull complete 
5f5781394e74: Pull complete 
4c758f5def4e: Pull complete 
cec232b6c8e8: Pull complete 
Digest: sha256:473744e71c112bb20fe897a3e0d985a6cb8aed98b6a9ae1461df5cf36214cb11
Status: Downloaded newer image for swift:5.7.0-jammy
root@8486157fd10c:/src# swift build
[1/1] Planning buildCompiling plugin MakeVapidKeyPlugin...
Compiling plugin Swift-DocC...
Compiling plugin Swift-DocC Preview...
Building for debugging...
/src/.build/checkouts/swift-http-types/Sources/HTTPTypes/HTTPFields.swift:165:68: error: cannot convert value of type 'String' to expected argument type 'String.Element' (aka 'Character')
                    self[fields: name] = newValue.split(separator: "; ", omittingEmptySubsequences: false).map {
                                                                   ^
/src/.build/checkouts/swift-http-types/Sources/HTTPTypes/HTTPFields.swift:165:68: error: cannot convert value of type 'String' to expected argument type 'String.Element' (aka 'Character')
                    self[fields: name] = newValue.split(separator: "; ", omittingEmptySubsequences: false).map {
                                                                   ^
[33/310] Compiling x_x509.c
```

--- a/Package.swift
+++ b/Package.swift
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.7
+// swift-tools-version: 5.7.1
 
 import PackageDescription
 

